channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/JaicpConnector.kt

```diff
@@ -31,10 +33,10 @@ abstract class JaicpConnector(
     val accessToken: String,
     val url: String,
     httpClient: HttpClient,
-    executorThreadPoolSize: Int = DEFAULT_REQUEST_EXECUTOR_THREAD_POOL_SIZE,
+    executor: Executor
```

Comment:
Let's make this backward-compatible, as constructors of connectors are used widely by users.
Let's just make the new constructor primary and the old one secondary.
The same remark is for JaicpPollingConnector and JaicpWebhookConnector.